napari_aicssegmentation/view/workflow_steps_view.py

```diff
@@ -116,15 +125,54 @@ def _add_workflow_steps(self, category: WorkflowStepCategory):
 
         self.layout.addSpacing(10)
 
+    def _add_bottom_buttons(self):
+        layout = QHBoxLayout()
+        layout.setSpacing(5)
+
+        btn_close_workflow = QPushButton("Close workflow")
+        btn_close_workflow.setFixedWidth(120)
+        btn_close_workflow.clicked.connect(self._btn_close_clicked)
+
+        btn_run_all = QPushButton("Run all")
+        btn_run_all.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Preferred)
+        btn_run_all.clicked.connect(self._btn_run_all_clicked)
+
+        layout.addWidget(btn_close_workflow)
+        layout.addWidget(btn_run_all)
+
+        self.layout.addLayout(layout)
+
     #####################################################################
     # Event handlers
     #####################################################################
 
-    def _btn_back_clicked(self, checked: bool):
-        self._controller.navigate_back()
-
     def _btn_info_clicked(self, checked: bool):
-        self.diagram = QLabel()
         diagram_path = str(Directories.get_assets_dir() / "workflow_diagrams/sec61b_1.png")
         self.diagram.setPixmap(QPixmap(diagram_path))
         self.diagram.show()
+
+    def _btn_close_clicked(self, checked: bool):
+        prompt = (
+            "<span>You are closing an in-progress Allen Cell & Structure Segmenter plugin workflow to return "
+            "to the Workflow Selection screen.&nbsp;Your progress in this workflow will be lost.</span>"
+        )
+
+        self.confirmation_modal.setModal(True)
+        self.confirmation_modal.setIcon(QMessageBox.Warning)
+        self.confirmation_modal.setText(f"Workflow: {self.workflow.name}")
+        self.confirmation_modal.setInformativeText(prompt)
+        self.confirmation_modal.setStandardButtons(QMessageBox.Cancel)
+
+        if len(self.confirmation_modal.buttons()) < 2:
+            self.close_keep = self.confirmation_modal.addButton("Close workflow", QMessageBox.AcceptRole)
+
+        self.confirmation_modal.exec()
+        self._handle_modal_input(self.confirmation_modal.clickedButton())
+
+    def _handle_modal_input(self, input):
+        if input == self.close_keep:
+            self._controller.reset_model()
```

Comment:
To keep responsibilities clear, these 2 calls should be merged into a single call to the controller, which could be named for example `close_workflow()`. From the view's perspective, we're just asking for a single action to be handed. The controller in return is responsible internally for resetting the model and initiating a navigation. And you can quickly see that once we add the complexity of having both "keep layers" and "don't keep layers", this would simply become `IWorkflowStepsController.close_workflow(keep_layers:bool)`